content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
 ---
-title: 'Deploy to AWS Lambda'
-metaTitle: 'Deploy to AWS Lambda'
-metaDescription: 'Learn how to deploy Node.js and TypeScript applications based on Prisma Client to AWS Lambda.'
+title: 'Deploy to AWS Lambda with Serverless Framework'
+metaTitle: 'Deploy your Serverless Framework project with Prisma'
+metaDescription: 'Learn how to avoid potential pitfalls when deploying your Serverless Framework project using Prisma'
 ---
 
 <TopBlock>
 
-This guide explains how to set up and deploy a serverless Node.js REST API to [AWS Lambda](https://aws.amazon.com/lambda/) with the [Serverless Framework](https://www.serverless.com/framework/).
-
-AWS Lambda is part of the AWS cloud platform. You can use it to run your code in a serverless environment. To deploy a REST API to AWS Lambda, you must also use [S3](https://aws.amazon.com/s3/) to host the files and [API Gateway](https://aws.amazon.com/api-gateway/) to expose the API over HTTP.
-
-The Serverless Framework simplifies deployment to Lambda. It provides a CLI that helps with workflow automation and AWS resource provisioning.
-
-This guide uses the [Prisma AWS example](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/aws-lambda), which has example REST endpoints preconfigured as serverless functions. The Prisma AWS example uses Prisma Client to fetch, create, and delete records from a database. Each function represents a REST resource endpoint and uses Prisma Client to handle database operations against a PostgreSQL database. In this guide, we explain how to host your database on Supabase, but you can use a platform of your choice.
-
-<details><summary>If you deploy GraphQL servers to AWS Lambda</summary>
-
-This example uses REST, but the same principles apply to a GraphQL server.<br /><br />The main difference is that you typically only need a single function to serve a
-GraphQL API. In that function, set the `context.callbackWaitsForEmptyEventLoop` of the [AWS Lambda Context Object](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-context.html) to `false` as follows:
-
-```js
-exports.server = (event, context, cb) => {
-  // Set to false to send the response right away when
-  // the callback executes. Don't wait for the Node.js
-  // event loop to be empty.
-  context.callbackWaitsForEmptyEventLoop = false
-
-  return lambda.graphqlHandler(event, context, cb)
-}
-```
-
-</details>
+This guide explains how to avoid common issues when deploying a [Serverless Framework](https://www.serverless.com/framework/) project with Prisma. For a guide on how to get started with Serverless Framework, we recommend [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
```

Comment:
```suggestion
This guide explains how to avoid common issues when deploying a project using Prisma with [Serverless Framework](https://www.serverless.com/framework/). For a guide on how to get started with Serverless Framework, we recommend [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
```